sample/mpp-library/src/commonTest/kotlin/ProfileApiTest.kt

```diff
@@ -34,6 +34,80 @@ class ProfileApiTest {
         }
     }
 
+    @Test
+    fun `test return map from api call`() {
+        val testResponse = """
+            {
+                "user1" : {
+                    "name" : "user1"
+                },
+                "user2" : {
+                    "name" : "user2"
+                }
+            }
+        """.trimIndent()
+
+        val httpClient = createMockClient {
+            respondOk(content = testResponse)
+        }
+        val defaultApi = DefaultApi(httpClient = httpClient, json = json)
+
+        val result = runBlocking {
+            defaultApi.apiProfileGetMap()
+        }
+
+        assertTrue { result["user1"]!!.name == "user1" }
+        assertTrue { result["user2"]!!.name == "user2" }
+    }
+
+    @Test
+    fun `test return set from api call`() {
+        val testResponse = """
+            {
+                "values" : ["str1", "str2"]
+            }
+        """.trimIndent()
+
+        val httpClient = createMockClient {
+            respondOk(content = testResponse)
+        }
+        val defaultApi = DefaultApi(httpClient = httpClient, json = json)
+
+        val result = runBlocking {
+            defaultApi.apiProfileGetSet()
+        }
+
+        assertNotNull(result.values)
+        assertTrue { result.values!!.size == 2 }
+        assertTrue { result.values!!.contains("str1") && result.values!!.contains("str2") }
+    }
+
+    @Test
+    fun `test request body with set`() {
+        val testResponse = """
+            [{"id" : "10"}]
+        """.trimIndent()
+
+        val httpClient = createMockClient {
+            respondOk(content = testResponse)
+        }
+        val profileApi = ProfileApi(httpClient = httpClient, json = json)
+
+        val result = runBlocking {
+            profileApi.profileActivityCreate(
+                userId = "1",
+                unitId = 1,
+                unitActivitySetItem = setOf(
+                    UnitActivitySetItem(activityRid = "1"),
+                    UnitActivitySetItem(activityRid = "2")
+                )
+            )
+        }
+
+        assertTrue { result.isNotEmpty() }
+        assertTrue { result.first().id == "10" }
```

Comment:
here better will be:
```
assertEquals(
    expected = setOf(Item(id = "10")),
    actual = result
)
```
but for now - ok

Reply:
Fixed